packages/instant/src/util/buy_quote_updater.ts

```diff
@@ -16,12 +16,12 @@ export const buyQuoteUpdater = {
         dispatch: Dispatch<Action>,
         asset: ERC20Asset,
         assetUnitAmount: BigNumber,
-        setPending = true,
+        options: { setPending: boolean; dispatchErrors: boolean },
         affiliateInfo?: AffiliateInfo,
```

Comment:
nit: `affiliateInfo` should also be part of options?

Reply:
updated with 33a156d